package.json

```diff
@@ -48,13 +48,17 @@
     "pre-commit": "lint-staged && npm run lint",
     "build:templates": "gulp templates",
     "start": "npm run build:templates && webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
-    "test": "karma start karma.config.js",
+    "lint": "eslint .",
+    "test:unit": "npm run lint && karma start karma.config.js",
+    "watch:templates": "gulp watch",
     "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
     "start:prod-local": "cd dist && http-server -p 4200 -c-1 -b",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
-    "lint": "eslint .",
+    "test:e2e": "npm run lint && playwright test",
```

Comment:
Instead of incorporating `lint` into both `test:unit` and `test:e2e` (which are basically unrelated), I think it would be better to have a `test` script that runs all three:

```json
"lint": "eslint .",
"test:unit": "karma start karma.config.js",
"test:e2e": "playwright test",
"test": "npm run lint && npm run test:unit && npm run test:e2e",
```

Reply:
Good idea! Fixed